apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -446,13 +446,22 @@ export class ModalPage {
     await this.page.getByTestId('w3m-wallet-guide-get-started').click()
   }
 
-  async promptSiwe() {
-    const siweSign = this.page.getByTestId('w3m-connecting-siwe-sign')
-    await expect(siweSign, 'Siwe prompt sign button should be visible').toBeVisible({
-      timeout: 10_000
-    })
-    await expect(siweSign, 'Siwe prompt sign button should be enabled').toBeEnabled()
-    await siweSign.click()
+  async promptSiwe(cancel = false) {
```

Comment:
Maybe we could have object param which make things easier to understand when using this function:

```suggestion
  async promptSiwe({ cancel }: TypeDef) {
  ...
  promptSiwe({ cancel: true })
```